### DISCUSSION TITLE
try to connect to slack but got This page is taking too long to load

### DISCUSSION THREAD

**Discussion by shehrozkapoor** [community/community#37057] [Other Feature Feedback, Questions, & Ideas]
![Screenshot 2022-10-24 at 1 21 37 AM](https://user-images.githubusercontent.com/59894229/197416407-ee3baf76-67e1-4ed0-af1c-f63659a125c2.png)

i got the same issue when i was try to connect codemagic for CI/CD

**Comment by ossanna16** [comment]
Hi there @shehrozkapoor and welcome to our community 😄. Thanks so much for asking a great question!
